### PR TITLE
Add a property to the config file for RackHD public IP

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
https://github.com/RackHD/RackHD/pull/741

**Comment by tannoa2** [comment]
https://github.com/RackHD/RackHD/pull/741

**Comment by tannoa2** [comment]
@RackHD/veyron @RackHD/corecommitters

File: packer/ansible/roles/monorail/files/config.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "amqp": "amqp://localhost",
     "apiServerAddress": "172.31.128.1",
+    "rackhdPublicId": null,
```
Comment (anhou) [inline packer/ansible/roles/monorail/files/config.json]: confused about 'Id'  with 'publich IP' in the PR title

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Pull request merged**